biotracks/cmso.py

```diff
@@ -39,3 +39,6 @@
 OBJECT_ID = "cmso_object_id"
 LINK_ID = "cmso_link_id"
 TRACK_ID = "cmso_track_id"
+
+SPATIAL_UNIT = "cmso_spatial_unit"
```

Comment:
I agree it's not consistent but both `SPACE_UNIT` and `TEMPORAL_UNIT` sounded weird to me when I thought about it. If we want to fix this I'm more inclined towards `SPACE/TIME` rather than `SPATIAL/TEMPORAL`.